index.html

```diff
@@ -684,8 +684,8 @@ <h2 class="wow fadeInLeftBig">Contact Us</h2>
             <div class="col-lg-3 col-md-3 col-sm-6">
               <div class="single_contact_feaured wow fadeInUp">
                 <i class="fa fa-envelope-o"></i>
-                <h4>Email Address</h4>
-                <p>singlepro@gmail.com</p>
+               <h4>Email Address</h4>
+                <p>admin@nitsurat.acm.org</p>

```

Comment:
I think this is also included in one of the issues I have filed. Please try fix complete issue. I can't merge the pieces of different issues.